projects/RabbitMQ.Client/Impl/RabbitMQActivitySource.cs

```diff
@@ -85,12 +88,12 @@ public static class RabbitMQActivitySource
             }
 
             Activity? activity = s_subscriberSource.StartRabbitMQActivity(
-                UseRoutingKeyAsOperationName ? $"{queue} receive" : "receive",
+                UseRoutingKeyAsOperationName ? $"{queue} {MessagingOperationTypeReceive}" : MessagingOperationTypeReceive,
```

Comment:
Is this activity created for `BasicGet` ? If so, let's pick a rabbitMQ-friendly operation name for it. 

See my comment on publish.

Also, the order has changed - it's now `{operation_name} {destination}`